CHANGES.rst

```diff
@@ -11,6 +11,8 @@ Unreleased
   ``None``.
 * Parse URLs with trailing slashes and no ``'comments'`` element when creating
   :class:`.Submission` objects.
+* Raise an exception when a subreddit URL is provided to the
```

Comment:
I don't think we need to actually add this as this bugfix only fixes things that were broken since the last release. I.e., they haven't been released yet.